tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java

```diff
@@ -57,8 +68,38 @@ public TesCreateTaskResponse submitTesTask(TesTask body) {
     }
 
     @Override
-    public TesListTasksResponse listTesTask() {
-        return new TesListTasksResponse();
+    public TesListTasksResponse listTesTask(String namePrefix, Long pageSize, String pageToken, TaskView view) {
+        TesListTasksResponse tesListTasksResponse = new TesListTasksResponse();
+        List<PipelineRun> pipelineRunList;
```

Comment:
try to rewrite it with `?:` operator and make it final

Reply:
refactored with ?: operator, removed local var